tools/releaseTools.psm1

```diff
@@ -246,32 +251,148 @@ function Get-NewOfficalPackage
         $packages=$csprojXml.Project.ItemGroup.PackageReference
 
         # check to see if there is a newer package for each refernce
-        foreach($package in $packages)
+        foreach ($package in $packages)
         {
             # Get the name of the package
             $name = $package.Include
 
             # don't pull 'Microsoft.Management.Infrastructure' from nuget
-            if($name -and $name -ne 'Microsoft.Management.Infrastructure')
+            if ($name -and $name -ne 'Microsoft.Management.Infrastructure')
```

Comment:
MMI package is on nuget.org.. we should check for that too.

https://www.nuget.org/packages/Microsoft.Management.Infrastructure/